src/pages/ServiceGraph/SummaryPanelEdge.tsx

```diff
@@ -217,14 +232,19 @@ export default class SummaryPanelEdge extends React.Component<SummaryPanelPropTy
         });
       })
       .catch(error => {
-        if (!this._isMounted) {
-          console.log('SummaryPanelEdge: Ignore fetch error, component not mounted.');
+        if (error.isCancelled) {
```

Comment:
I think this is actually wrong, as you are using 'isCanceled' in the utils.

Reply:
Ops, yes. Good catch.
All references changed.